codecov.yml

```diff
@@ -0,0 +1 @@
+comment: false
```

Comment:
```suggestion
comment:
   layout: diff
   behaviour: once
```

Reply:
According to the [documentation](https://docs.codecov.com/docs/pull-request-comments#behavior), it is supposed to push one comment and update it if there is any change. Also, it should only display the diff.

WDYT?

Reply:
You are right, it will avoid being too noisy.

Reply:
> Shouldn't `require_changes` be `true`?

:facepalm: Thanks.